src/utils/functions.js

```diff
@@ -0,0 +1,9 @@
+// Render loading
+export function renderLoading(isLoading, buttonElement, loadingText) {
+
+    if (isLoading) {
+        buttonElement.textContent = loadingText;
+    } else {
+        return buttonElement;
```

Comment:
You should change button text to the initial here. To fix that you can rename this method to `setSubmitButtonText`, so it will just change the text

```js
export function setSubmitButtonText(buttonElement, text) {
  buttonElement.textContent = text;


// when want to show loading
setSubmitButtonText(buttonElement, 'Loading..')
// when want to reset the text
setSubmitButtonText(buttonElement, 'Save')
```

Reply:
Neat! I've updated the function accordingly. I really appreciate the detailed explanation!

**Quick Question**: The project instruction states the loading status to be "Saving...". I understand it's a wording preference but it'd be great to get clarification. Should I be using "Loading..." for all loading status or custom loading status, such as "Saving...", "Creating..." or "Deleting..."?

Reply:
Great! Thank you!

You can use any texts, it depends only on your preference, the main thing is that when loading they mean the process

Reply:
Gotcha! Thanks for clarifying!